versions/v6/functions/config.php

```diff
@@ -3,7 +3,8 @@
 
 
 array_push( Config::$styles,
-	array('name' => 'style-css', 'src' => THE_POST_VERSION_URL . '/static/css/style.min.css')
+	array('name' => 'style-css', 'src' => THE_POST_VERSION_URL . '/static/css/style.min.css'),
+	array('name' => 'style-fa', 'src' => THE_POST_VERSION_URL . '/static/css/font-awesome-5.min.css')
```

Comment:
We probably want to make sure fontawesome loads prior to style.min.css; flipping these two lines should accomplish that